libs/accelerometer/accelhw.cpp

```diff
@@ -11,11 +11,11 @@
 #include "CodalDmesg.h"
 
 #ifndef PXT_DEFAULT_ACCELEROMETER
-#define PXT_DEFAULT_ACCELEROMETER ACCELEROMETER_TYPE_LIS3DH
+#define PXT_DEFAULT_ACCELEROMETER -1
 #endif
 
 #ifndef PXT_SUPPORT_LIS3DH
-#define PXT_SUPPORT_LIS3DH 1
+#define PXT_SUPPORT_LIS3DH 0
```

Comment:
Will need to define PXT_SUPPORT_LIS3DH to 1 in pxt-adafruit

Reply:
put it back... and to be sure: https://github.com/Microsoft/pxt-adafruit/pull/999